packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -309,7 +309,7 @@ export class ForkChoice implements IForkChoice {
     blockDelaySec: number,
     currentSlot: Slot,
     executionStatus: MaybeValidExecutionStatus
-  ): void {
+  ): ProtoBlock {
```

Comment:
Make onBlock return the blockSummary to prevent having to consider in the forkchoice that the new block is not found in the fork-choice